admin-app/image.go

```diff
@@ -99,3 +131,21 @@ func postImageHandler(app_settings common.AppSettings, database database.Databas
 		})
 	}
 }
+
+func checkContentTypeMatchesData(file_header *multipart.FileHeader) (string, error) {
```

Comment:
Function should be called something like `getContentType(...)`

Reply:
I agree. I refactored it. Previously it returned a `boolean` which indicated if the check was successful or not.